Sources/MapboxSearch/PublicAPI/DefaultLocationProvider.swift

```diff
@@ -109,7 +109,7 @@ extension DefaultLocationProvider: LocationProvider {
 }
 
 private class DefaultLocationManagerDelegate: NSObject, CLLocationManagerDelegate {
-    unowned var locationProvider: DefaultLocationProvider
+    unowned(unsafe) var locationProvider: DefaultLocationProvider
```

Comment:
Could you please specify why this change is needed? Actually, I don't see why `unowned` is used here, can we replace it to usual `var` property?